File: RecargaPlus/Controller/objects/Recarga.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Controller.objects
{
    public class Recarga
    {
        public int ID { get; set; }
        public String Numero { get; set; }
        public Colaborador Colaborador { get; set; }
        public Bonificacion Bonificacion { get; set; }
        public Compania Compania { get; set; }
        public Monto Monto { get; set; }
    }
}

File: Andtoid/app/src/main/java/Objects/Recarga.java
package Objects;

public class Recarga {
    private int id;
    private String numero;
    public Colaborador colaborador;
    public Bonificacion bonificacion;
    public Compania compania;
    public Monto monto;
    public Recarga(){
        this.colaborador = new Colaborador();
        this.bonificacion = new Bonificacion();
        this.compania = new Compania();
        this.monto = new Monto();
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNumero() {
        return numero;
    }

    public void setNumero(String numero) {
        this.numero = numero;
    }
}
